# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h
# Commit: 604c4c52574a
# Full Hash: 604c4c52574ae48f4f168eb999bc0af15dd5e0d6
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-04-15 21:21:06
# Regressor Bug: 1899503
# File Overlap Count: 2
# Description:
#   Bug 1899503 - P2 PRemoteWorkerDebuggerMangaer binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebuggerManager binding.
#   
#   PRemoteWorkerDebuggerManager is the IPC that handles the registration/unregistration of remote debuggers.
# ==============================================================================

diff -r 77eda17d0795 -r 604c4c52574a dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h
--- a/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h	Tue Apr 15 09:47:38 2025 +0000
+++ b/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h	Tue Apr 15 09:47:39 2025 +0000
@@ -19,6 +19,9 @@
   NS_INLINE_DECL_THREADSAFE_REFCOUNTING(RemoteWorkerDebuggerManagerParent,
                                         final)
 
+  static RefPtr<RemoteWorkerDebuggerManagerParent> CreateForProcess(
+      mozilla::ipc::Endpoint<PRemoteWorkerDebuggerManagerChild>* aChildEp);
+
   RemoteWorkerDebuggerManagerParent();
 
   mozilla::ipc::IPCResult RecvRegister(